db/models.py

```diff
@@ -1,5 +1,11 @@
+import datetime
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+import settings
```

Comment:
it's better to use
`from django.conf import settings` because it defines correct settings(as there may be a folder named settings)